### DISCUSSION TITLE
Auto fan curve - non integer Step, Response & Deadband values

### DISCUSSION THREAD

**Discussion by Stego72** [Rem0o/FanControl.Releases#1451] [Ideas]
Hi

Would it be possible to have decimal numbers for the Step, Response time and Deadband values for the Auto Fan Control ?

The current integer values really makes fine-tuning difficult - especially wrt Step and Response time

Thx for a great project

**Comment by Rem0o** [comment]
Step and deadband have decimal, you just need to type it in. Response time can't have decimal, since the internal clock of the software works at 1 hz.

**Comment by Stego72** [comment]
Thanks 😊
